benchmarks/execute/src/main.rs

```diff
@@ -60,6 +70,39 @@ struct Cli {
     verbose: bool,
 }
 
+#[derive(Clone, Debug, VmConfig, Serialize, Deserialize)]
+pub struct ExecuteConfig {
```

Comment:
how come this is dup of `execute.rs`?

Reply:
Yeah, not ideal. Let me create a common file